themes/socialbase/templates/profile/profile--profile--compact_teaser.html.twig

```diff
@@ -20,6 +20,11 @@
  */
 #}
 {{ attach_library('socialbase/teaser') }}
+
+{% if style == 'sky' %}
+  {{ attach_library('socialblue/teaser--sky') }}
```

Comment:
We should use a libraries definition alter hook to add our files to the right library if the sky flavour is enabled. Especially since this code references socialblue from socialbase which shouldn't happen.